### DISCUSSION TITLE
Soil permeability

### DISCUSSION THREAD

**Discussion by muhammedfurkanyilmaz** [KratosMultiphysics/Kratos#12022] [Q&A]
Dear all,
I'm trying to model coupled dynamic consolidation analysis, but I'm having trouble with the permeability parameters units. I don't understand what units Kratos used. Because when I input in m/sec results in terms of excess pore pressures are quite low when compared to the Plaxis. Besides that, the permeability parameters are indicated as 0.001 day/m in tutorial-4 documents but logged as 1.17982E-15 into the program in the test analysis file. What's the point here? Any help will be appreciated.
Thanks.

**Comment by loumalouomega** [comment]
A priori everything between Kratos is IS. May be the implementation which hardcoded the units.

**Reply by muhammedfurkanyilmaz** [reply]
how can I overcome that?

**Reply by loumalouomega** [reply]
I have no idea @KratosMultiphysics/geomechanics ?

**Reply by muhammedfurkanyilmaz** [reply]
Thanks. (Yes, Geomechanics.)

**Reply by avdg81** [reply]
Dear Muhammed, first of all, thank you for contacting us! I am sending you this reply on behalf of the @KratosMultiphysics/geomechanics team. We (the developers of the GeoMechanicsApplication at Deltares) have briefly discussed your question this morning. However, to assist you properly we would like to receive some additional input from you. Would you be so kind to answer the following questions?

1. Could you please confirm that you are using the GeoMechanicsApplication, since that was not explicitly mentioned in your question? If you are using some other Kratos application (e.g. the PoroMechanicsApplication) then you'd better ask the maintainers of that particular application.
2. Could you please indicate where you got your input files from? You mentioned "tutorial-4 documents" in your question, but it is unclear to us which one you mean. Could you please indicate more precisely which tutorial you are referring to (e.g. by sharing a link to it)?
3. If you have made any changes to the input files in order to run the simulation, could you please share those with us as well?

When we have received sufficient information, we will try to reproduce your issue at our end.

Thank you in advance for your response.

**Comment by muhammedfurkanyilmaz** [comment]
Dear Anne, thanks for your reply. 

1. I'm using the Geomechanics application. 
2. I downloaded the input files from the Kratos/Multiphysics repository when I first installed the software. Here is the link for the related test files: https://github.com/KratosMultiphysics/Kratos/tree/master/applications/GeoMechanicsApplication/tests/1D-Consolidation_all_stages
I have validated the result of this test by Terzaghis 1D consolidation theory for the 0.001 m/day permeability. But in the test files, permeability is taken as 1.17982E-15.
3. I've only changed the height of the column from 1m to 10m and the element type from 2D6N to 2D4N. All boundaries and loading conditions remained the same. 

Regards
MFY

**Reply by avdg81** [reply]
Dear Muhammed,

Thank you for your swift reply. We have now sufficient information to investigate the issue. As soon as we understand the problem, we will inform you.

Kind regards,
Anne

**Reply by avdg81** [reply]
Dear Muhammed,

In [this PR](https://github.com/KratosMultiphysics/Kratos/pull/12041) we have simplified and cleaned up the input files of the test, and we have removed the mechanism that would skip the test. Furthermore, we have written some documentation to make clear what the test does and how we obtained the reference results. Could you please have a look and see whether those changes already answer your question? If not, could you please indicate as accurately as possible which parts need further elaboration? Thanks in advance for your feedback.

Kind regards,
Anne

**Reply by muhammedfurkanyilmaz** [reply]
Dear Anne, 

Thanks, now everything is quite clear for me.

Regards.
MFY

**Comment by rubenzorrilla** [comment]
Closing as it seems to be solved.
